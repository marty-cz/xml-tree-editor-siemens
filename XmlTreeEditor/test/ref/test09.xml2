<?xml version="1.0" encoding="ISO8859-2" ?>
<CATALOG>
  <CD ID="cd01">
    <TITLE>Empire Burlesque</TITLE>
    <ARTIST>Bob Dylan</ARTIST>
    <COUNTRY>USA</COUNTRY>
    <COMPANY>Columbia</COMPANY>
    <PRICE>10.90</PRICE>
    <YEARxxxxxxx>1985</YEAR>
  </CD>
  <CD ID="cd02">
    <TITLE>Hide your heart</TITLE>
    <ARTIST>Bonnie Tylor</ARTIST>
    <COUNTRY>UK</COUNTRY>
    <COMPANY>CBS Records</COMPANY>
    <PRICE>9.90</PRICE>
    <YEAR>1988</YEAR>
  </CD>
</CATALOG> 