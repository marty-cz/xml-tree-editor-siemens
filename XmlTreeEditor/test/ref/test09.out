Error: test09.xml: Does not correspond with schema "schema01.xsd": cvc-complex-type.2.4.a: Invalid content was found starting with element 'YEARxxxxxxx'. One of '{YEAR}' is expected.
